src/components/inputs/date-time-input/date-time-input.visualspec.js

```diff
@@ -0,0 +1,16 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { widths } from '../../../../test/percy/widths';
+
+describe('DateTimeIpnut', () => {
+  it('Default', async () => {
+    await page.goto('http://localhost:3001/date-time-input');
+    await expect(page).toMatch('minimal');
+    await percySnapshot(page, 'DateTimeInput', { widths });
+  });
+  it('Open', async () => {
+    await page.goto('http://localhost:3001/date-time-input-open');
+    await page.click('#date-time-input');
```

Comment:
here's an example of interacting with a component before taking it's picture

Reply:
Is this a case for data-testid rathet than an id?